File: JavaScript/6-break.js
'use strict';

const flag = false;

label1: for (let i = 0; i < 10; i++) {
  console.log('Hello');
  if (flag) break label1;
  console.log('World');
  label2: for (let j = 0; j < 10; j++) {
    console.log('There');
    break label1;
    console.log('Bye');
  }
  console.log('End');
}

File: C++/6-break.cpp
#include "stdafx.h"
#include <iostream>
#include "conio.h"

using namespace std;

void Break()
{
	bool flag = true;
	while (true)
	{
		cout << "Hello ";
		if (flag) break;
		cout << "World!";
	}

	cout << "There!";

	_getch();
}